### ISSUE TITLE
Main window is not appearing

### ISSUE THREAD

**Issue by pglira** [Run1e/Columbus#1]
I'm using the last version of columbus, but the main window is not appearing, neither with the shortcut, nor by choosing "show columbus" in the menu. However, the task switcher works fine and appears.

**Comment by Run1e** [comment]
Are you running the script as a ahk file or have you downloaded the exe?
If you're running as an ahk file, please make sure your version of [AutoHotkey is up to date](http://ahkscript.org/download/ahk-install.exe). Or try using the [exe](http://runie.me/Columbus/Columbus.exe).

If the problem persists, I'll write a test script for you to try.

**Comment by pglira** [comment]
I have the latest version of AHK (v1.1.22.07). I tried both, the exe and the ahk file - without success. Im using Win10 (x64).

**Comment by Run1e** [comment]
The window might be off the screen. Have Columbus running and launch this script (in a new ahk file): http://p.ahkscript.org/?p=d404d837

It should move your window to the top-left (x50, y50).

**Comment by pglira** [comment]
I have running the executable of columbus, but when i start the proposed script it tells me:
![2015-10-16 21_07_54](https://cloud.githubusercontent.com/assets/14214284/10550918/05681988-744a-11e5-9659-6500953b005a.png)

**Comment by Run1e** [comment]
Try running the ahk version of Columbus found [here](http://runie.me/Columbus/Columbus.ahk)

**Comment by pglira** [comment]
I'm sorry, thanks for your help, but it really won't appear. I tried it also on another computer (win7, x64).

**Comment by Run1e** [comment]
Try running this. It moves the window to your current mouse pos, and displays a msgbox telling whether the window exists. If this doesn't work then there's a win10 bug. You could also try running Columbus as administrator.

http://p.ahkscript.org/?p=f7ca0ec2

**Comment by pglira** [comment]
Thank you! First it shows this:
![2015-10-17 21_06_50](https://cloud.githubusercontent.com/assets/14214284/10560692/13357fb4-7513-11e5-945d-560f279c97e6.png)

Then this:
![2015-10-17 21_06_56](https://cloud.githubusercontent.com/assets/14214284/10560688/10072554-7513-11e5-8bc1-4655f996be5b.png)

Edit: I retried Columbus v0.50 and the main window appears in the bottom right corner. However, I can't see what I'm typing:
![2015-10-17 21_16_19](https://cloud.githubusercontent.com/assets/14214284/10560732/51f59e04-7514-11e5-9d70-6d95068837f0.png)
(This screenshot shows the bottom right corner of the screen)

**Comment by Run1e** [comment]
Ok. I've talked to someone else with windows 10 and it seems like there is a bug/complication with plugins. We managed to get it to work if you run the plugin as administrator. Please try this.

Edit: Just saw your edit on your comment. This is a weird problem you're having, and the weird window placement in v0.50 is probably related to your current problem. Are there any special traits to the monitor/vfx card you're using? Are you on a 4K screen maybe?

**Comment by pglira** [comment]
I tried to run columbus.exe in admin mode on two different pcs: one with win10x64, the other with win7x64. On both pcs the windows is not appearing. I don't know how to start the ahk file (columbus.ahk) in admin mode.

**Comment by Run1e** [comment]
Does the previous window that had focus lose focus when you press CTRL+ALT+P? If so the window does pop up and it's a placement issue.

Try these steps:
Close Columbus (exit in the traymenu).
Go to Program Files (x86) and delete the Columbus folder.
Start Columbus again.

**Comment by Run1e** [comment]
If the previous window does lose focus (something that indicates that the Columbus window is in focus), you can also type in some commands to move the window.

Try this:
CTRL+ALT+P so that the Columbus window appear to be focused
Type in: /set Pos.X 5
Press enter
Type in: /set Pos.Y 5
Restart Columbus.

This should put Columbus in the top left corner of your main screen.

**Comment by pglira** [comment]
Ok, after running the set commands I can see the windows. Thanks.

**Comment by Run1e** [comment]
Cool! It's nice to hear you got it working and it's usable now. I assume you know you can use the /move command to move the window now.

Could you please run this for me, and tell me what it outputs? I would also appreciate it if you could tell me your actual monitor resolution.

```
WinGetPos,,,,th,ahk_class Shell_TrayWnd
msgbox % "Screen`nH: " A_ScreenHeight "`nW: " A_ScreenWidth "`n`nColumbus`nX: " A_ScreenWidth-502 "`nY: " A_ScreenHeight-357-th "`n`nTrayHeight: " th
```

**Comment by pglira** [comment]
Output is:
![2015-10-19 10_04_07](https://cloud.githubusercontent.com/assets/14214284/10572453/d15b84b2-7648-11e5-9ae7-3e03289ae5e8.png)

On the computer I started the script above, the resolution is 1920x1200. On the other computer, on which columbus also didn't appear, the resolution is 1920x1080.

**Comment by Run1e** [comment]
Ok, that explains why the window is not appearing. It thinks your taskbar is 1200px high, when in reality it's about 30-40px. I'll figure out wtf is wrong. I'll just add a fail-safe to it when I get home later. Thanks for the debugging help.

**Comment by pglira** [comment]
Well, my taskbar is 1200px high:
![2015-10-19 11_44_57](https://cloud.githubusercontent.com/assets/14214284/10574541/e7943ebe-7656-11e5-8056-567edd188f6d.png)
I have it always on the left side of my screen. :-)

**Comment by Run1e** [comment]
Haha! Ok now I know how to fix this problem :) Thanks for the help, I'll update the software when I get home in 3-4 hours.

Edit: would be cool to know where you would like the window to be placed now.

**Comment by Run1e** [comment]
Bug has been fixed in v0.931, issue closed.
